### PR TITLE
Add roadmap document

### PR DESCRIPTION
**Pull request by carlossvg**
@razr I created a draft for the roadmap section. Please review the current and provide some feedback about missing tasks, how to distribute the tasks, deadlines, etc. Thanks!

### CODE REVIEW

**Comment by carlossvg** [comment]
The Concepts sections is probably a too large task. We may want to split it in different concepts so it's easier to prioritize and distribute the work.

**Comment by carlossvg** [comment]
@razr I would add an "Tutorial" section to the documentation to explain how to run some tests and demos with real-time settings in ROS 2. I would follow a similar format than this document https://docs.ros.org/en/foxy/Tutorials/Real-Time-Programming.html. 

I proposed the following tutorials:
- How to run pendulum_control with real-time settings (https://github.com/ros2/demos/tree/master/pendulum_control)
- How to run performance test with real-time settings
- How to run the pendulum demo (https://github.com/ros2-realtime-demo/pendulum)
- How to run Autoware.Auto demo ( i.e: perception stack https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/issues/683#note_522250258)

File: source/Roadmap/Roadmap.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Roadmap
+
+This page describes planned work for the ROS 2 Real-Time Working Group.
+
+
+| **Task**                                                     | **Size** | **Owner** | **Expected Completion** |
+| ------------------------------------------------------------ | -------- | --------- | ----------------------- |
+| Documentation: Complete Linux real-time guide                | Small    |           | 2nd quarter 2021        |
+| Documentation: Add instruction, motivation and <br/> goals documents | Small    |           | 2nd quarter 2021        |
```
Comment (razr) [inline source/Roadmap/Roadmap.md]: I think we should follow the *One line per task* rule.  if these are different documents it is better to split this line.

File: source/Roadmap/Roadmap.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| Documentation: How to configure and run a ROS2 <br/> Real-Time application | Small    |           | 2nd quarter 2021        |
+| Documentation: Real-Time Buildfarm                           | Small    |           | 2nd quarter 2021        |
```
Comment (razr) [inline source/Roadmap/Roadmap.md]: is it a tutorial or howto? What kind of document is it? Probably we can provide a more detailed description?

File: source/Roadmap/Roadmap.md (hunk 1 above)
Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: I split this in two tasks

File: source/Roadmap/Roadmap.md (hunk 2 above)
Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: I split this in two tasks

**Comment by carlossvg** [comment]
@razr I split the concepts tasks in different documents

File: source/Roadmap/Roadmap.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| ------------------------------------------------------------ | -------- | --------- | ----------------------- |
+| Documentation: Complete Linux real-time guide                | Small    |           | 2nd quarter 2021        |
+| Documentation: Add introduction document                     | Small    |           | 2nd quarter 2021        |
+| Documentation: How to configure and run a ROS2 <br/> Real-Time application | Small    |           | 2nd quarter 2021        |
+| Documentation:  How to run run the build-farm <br/>real-time tests locally | Small    |           | 2nd quarter 2021        |
+| Documentation:  Build-farm setup description                 |          |           |                         |
+| Buidfarm: Implement infrastructure to run the <br/> ROS 2 real-time tests | Large    |           |                         |
+| Buidfarm: Provide hardware to deploy and run <br/> the ROS 2  real-time tests | Small    |           |                         |
+| Buidfarm: Improve existing tools to measure <br/> all ROS 2  different executors | Medium   |           |                         |
```
Comment (carlossvg) [inline source/Roadmap/Roadmap.md]: Add item to define RT requirements (KPIs)

Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: Add item for "Carry out performance test measurements for ROS 2 stack" https://github.com/ros-realtime/community/issues/7

Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: Add Related issues, PRs column

Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: Add item for "Consolidate HW setting + OS setting"  https://github.com/ros-realtime/community/issues/13

**Comment by carlossvg** [comment]
Close https://github.com/ros-realtime/community/issues/12 with this PR

**Review by razr — APPROVED** [review]

File: source/Roadmap/Roadmap.md (hunk 3 above)
Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: Add tutorial for https://github.com/ros2/examples/tree/master/rclcpp/executors/cbg_executor

Comment (carlossvg) [reply source/Roadmap/Roadmap.md]: Separate tutorials in different sections like in micro-ros documentation: https://micro.ros.org//docs/tutorials/core/overview/

**Review by LanderU — APPROVED** [review]
LGTM!

**Pull request merged**